x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -232,62 +235,60 @@ const excludeLinks = (linkIds: SecurityPageName[]) => ({
   links: links.links?.filter((link) => !linkIds.includes(link.id)),
 });
 
-const getHostIsolationExceptionTotal = async (http: CoreStart['http']) => {
-  const hostIsolationExceptionsApiClientInstance =
-    HostIsolationExceptionsApiClient.getInstance(http);
-  const summaryResponse = await hostIsolationExceptionsApiClientInstance.summary();
-  return summaryResponse.total;
-};
-
 export const getManagementFilteredLinks = async (
   core: CoreStart,
   plugins: StartPlugins
 ): Promise<LinkItem> => {
   const fleetAuthz = plugins.fleet?.authz;
-  const { endpointRbacV1Enabled } = ExperimentalFeaturesService.get();
-  const hasPermissionsForSecuritySolution = calculatePermissionsFromCapabilities(
-    core.application.capabilities
-  );
+
+  const { endpointRbacEnabled, endpointRbacV1Enabled } = ExperimentalFeaturesService.get();
+  const isEndpointRbacEnabled = endpointRbacEnabled || endpointRbacV1Enabled;
+  const endpointPermissions = calculatePermissionsFromCapabilities(core.application.capabilities);
+
   const linksToExclude: SecurityPageName[] = [];
 
-  try {
-    const currentUserResponse = await plugins.security.authc.getCurrentUser();
-    const {
-      canReadActionsLogManagement,
-      canUnIsolateHost,
-      canIsolateHost,
-      canAccessEndpointManagement,
-    } = fleetAuthz
-      ? calculateEndpointAuthz(
-          licenseService,
-          fleetAuthz,
-          currentUserResponse.roles,
-          endpointRbacV1Enabled,
-          hasPermissionsForSecuritySolution
-        )
-      : getEndpointAuthzInitialState();
+  const currentUser = await plugins.security.authc.getCurrentUser();
 
-    if (!canReadActionsLogManagement) {
-      linksToExclude.push(SecurityPageName.responseActionsHistory);
-    }
+  const isPlatinumPlus = licenseService.isPlatinumPlus();
+  let hasHostIsolationExceptions: boolean = isPlatinumPlus;
 
-    if (!canIsolateHost && canUnIsolateHost) {
-      let shouldBeAbleToDeleteEntries: boolean;
-      try {
-        const hostExceptionCount = await getHostIsolationExceptionTotal(core.http);
-        // has an HIE entry and is a super user then set to TRUE
-        shouldBeAbleToDeleteEntries = hostExceptionCount !== 0 && canAccessEndpointManagement;
-      } catch {
-        shouldBeAbleToDeleteEntries = false;
-      }
+  // If not Platinum+ license and user has read permissions to security solution
+  // then check if Host Isolation Exceptions exist.
+  // *** IT IS IMPORTANT *** that  this HTTP call only be made if the user has access to the
+  // Lists plugin, else non-security solution users, especially when license is not Platinum,
+  // may see failed HTTP requests in the browser console. This is the reason that
+  // `hasKibanaPrivilege()` is used below.
+  if (
+    !isPlatinumPlus &&
+    fleetAuthz &&
+    hasKibanaPrivilege(
+      fleetAuthz,
+      isEndpointRbacEnabled,
```

Comment:
NIT: Would it be possible to move these two lines:
```
const { endpointRbacEnabled, endpointRbacV1Enabled } = ExperimentalFeaturesService.get();
const isEndpointRbacEnabled = endpointRbacEnabled || endpointRbacV1Enabled;
```
Inside the `hasKibanaPrivilege` function, instead of passing the value around as a parameter?

Reply:
yeah, that would make it cleaner @semd . I will look into it. Thanks for the feedback

Reply:
Actually, on second thought - I can't really make the change you suggested because `hasKibanaPrivilege()` is also used from the server side (via `calculateEndpointAuthz()`) and `ExperimentalFeaturesService` singleton is a UI side only.

Reply:
I see, all good then 👍